### ISSUE TITLE
After upgrade to 1.8.6 streaming doesn't work

### ISSUE THREAD

**Issue by mkrn** [pedroSG94/rtmp-rtsp-stream-client-java#646]
When streaming starts video camera preview hangs in connecting.. state. 
Same works on 1.8.5

Device: Samsung S10 Lite, Android 9

Code: 
```
private boolean prepareEncoders() {
        return rtmpCamera2.prepareVideo(1280, 720, VIDEO_FPS, 2000 * 1024, false, 2, CameraHelper.getCameraOrientation(this))
            && rtmpCamera2.prepareAudio();
    }

 if (prepareEncoders()) {
            try {
                statusLabel.setVisibility(View.VISIBLE);
                statusLabel.setText("Connecting...");
                statusLabel.setBackgroundColor(getResources().getColor(R.color.poor_signal));
                hintText.setVisibility(View.INVISIBLE);

                SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMdd_HHmmss", Locale.getDefault());
                currentDateAndTime = sdf.format(new Date());

                button.setBackgroundResource(R.drawable.stopbutton);
                rtmpCamera2.startStream(rtmpurl);

                videoFileName = "/" + getString(R.string.video_file_prefix) + currentDateAndTime + ".mp4";
                pathToFile = folder + videoFileName;
                Log.d(TAG, "Path to File: " + pathToFile);
                rtmpCamera2.startRecord(pathToFile);
```

Here's the output in LogCat:
```
2020-08-19 13:04:20.896 30324-30324/pro.eventlive D/ViewRootImpl@a6e7734[RTMPActivity]: ViewPostIme pointer 0
2020-08-19 13:04:21.061 30324-30324/pro.eventlive D/ViewRootImpl@a6e7734[RTMPActivity]: ViewPostIme pointer 1
2020-08-19 13:04:21.214 30324-30339/pro.eventlive I/pro.eventlive: NativeAlloc concurrent copying GC freed 340546(13MB) AllocSpace objects, 26(696KB) LOS objects, 64% free, 6652KB/18MB, paused 236us total 150.720ms
2020-08-19 13:04:21.275 30324-30341/pro.eventlive W/System: A resource failed to call release. 
2020-08-19 13:04:21.355 30324-30344/pro.eventlive I/CameraManagerGlobal: Camera 0 facing CAMERA_FACING_BACK state now CAMERA_STATE_IDLE for client pro.eventlive API Level 2
2020-08-19 13:04:21.417 30324-30344/pro.eventlive I/CameraManagerGlobal: Camera 0 facing CAMERA_FACING_BACK state now CAMERA_STATE_CLOSED for client pro.eventlive API Level 2
2020-08-19 13:04:21.462 30324-30324/pro.eventlive W/VideoCapabilities: Unsupported mime video/wvc1
2020-08-19 13:04:21.480 30324-30324/pro.eventlive W/Utils: could not parse long range '175-174'
2020-08-19 13:04:21.482 30324-30324/pro.eventlive W/AudioCapabilities: Unsupported mime audio/x-ima
2020-08-19 13:04:21.485 30324-30324/pro.eventlive W/AudioCapabilities: Unsupported mime audio/mpeg-L1
2020-08-19 13:04:21.486 30324-30324/pro.eventlive W/AudioCapabilities: Unsupported mime audio/mpeg-L2
2020-08-19 13:04:21.486 30324-30324/pro.eventlive W/AudioCapabilities: Unsupported mime audio/x-ms-wma
2020-08-19 13:04:21.533 30324-30324/pro.eventlive W/VideoCapabilities: Unsupported mime video/mp43
2020-08-19 13:04:21.535 30324-30324/pro.eventlive W/VideoCapabilities: Unrecognized profile/level 1/32 for video/mp4v-es
2020-08-19 13:04:21.535 30324-30324/pro.eventlive W/VideoCapabilities: Unrecognized profile/level 32768/2 for video/mp4v-es
2020-08-19 13:04:21.535 30324-30324/pro.eventlive W/VideoCapabilities: Unrecognized profile/level 32768/64 for video/mp4v-es
2020-08-19 13:04:21.537 30324-30324/pro.eventlive W/VideoCapabilities: Unsupported mime video/wvc1
2020-08-19 13:04:21.540 30324-30324/pro.eventlive W/VideoCapabilities: Unsupported mime video/x-ms-wmv7
2020-08-19 13:04:21.541 30324-30324/pro.eventlive W/VideoCapabilities: Unsupported mime video/x-ms-wmv8
2020-08-19 13:04:21.544 30324-30324/pro.eventlive I/VideoEncoder: VideoEncoder OMX.qcom.video.encoder.avc
2020-08-19 13:04:21.544 30324-30324/pro.eventlive I/VideoEncoder: Color supported: 2141391876
2020-08-19 13:04:21.544 30324-30324/pro.eventlive I/VideoEncoder: Color supported: 2130708361
2020-08-19 13:04:21.547 30324-30324/pro.eventlive I/ACodec:  [] Now uninitialized
2020-08-19 13:04:21.549 30324-30794/pro.eventlive I/ACodec: [] onAllocateComponent
2020-08-19 13:04:21.551 30324-30794/pro.eventlive I/OMXClient: IOmx service obtained
2020-08-19 13:04:21.671 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Loaded
2020-08-19 13:04:21.674 30324-30324/pro.eventlive I/VideoEncoder: Prepare video info: SURFACE, 1280x720
2020-08-19 13:04:21.681 30324-30794/pro.eventlive I/ACodec: app-pid(30324)
2020-08-19 13:04:21.683 30324-30794/pro.eventlive W/OMXUtils: do not know color format 0x7fa30c04 = 2141391876
2020-08-19 13:04:21.683 30324-30794/pro.eventlive W/OMXUtils: do not know color format 0x7f000789 = 2130708361
2020-08-19 13:04:21.701 30324-30794/pro.eventlive I/ACodec: app-name : pro.eventlive
2020-08-19 13:04:21.702 30324-30794/pro.eventlive I/ACodec: setupAVCEncoderParameters with [profile: Baseline] [level: Level31]
2020-08-19 13:04:21.704 30324-30794/pro.eventlive I/ACodec: Enable Perceptual Video Coding
2020-08-19 13:04:21.705 30324-30794/pro.eventlive I/ACodec: Success set IPB VideoMinQP(2/2/2) VideoMaxQP(50/50/50)
2020-08-19 13:04:21.705 30324-30794/pro.eventlive I/ACodec: reconfigEncoder4OtherApps
2020-08-19 13:04:21.711 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] cannot encode HDR static metadata. Ignoring.
2020-08-19 13:04:21.711 30324-30794/pro.eventlive I/ACodec: setupVideoEncoder succeeded
2020-08-19 13:04:21.711 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] configure, AMessage : AMessage(what = 'conf', target = 1) = {
      int32_t color-format = 2130708361
      int32_t i-frame-interval = 2
      string mime = "video/avc"
      int32_t width = 1280
      int32_t bitrate = 2048000
      int32_t max-input-size = 0
      int32_t frame-rate = 30
      int32_t height = 720
      int32_t encoder = 1
    }
2020-08-19 13:04:21.718 30324-30794/pro.eventlive W/OMXUtils: do not know color format 0x7f000789 = 2130708361
2020-08-19 13:04:21.769 30324-30324/pro.eventlive I/VideoEncoder: prepared
2020-08-19 13:04:21.805 30324-30324/pro.eventlive I/MicrophoneManager: Microphone created, 32000hz, Stereo
2020-08-19 13:04:21.848 30324-30805/pro.eventlive D/CCodec: allocate(c2.android.aac.encoder)
2020-08-19 13:04:21.850 30324-30805/pro.eventlive I/Codec2Client: Available Codec2 services: "software"
2020-08-19 13:04:21.851 30324-30805/pro.eventlive I/Codec2Client: Creating a Codec2 client to service "software"
2020-08-19 13:04:21.852 30324-30805/pro.eventlive I/Codec2Client: Client to Codec2 service "software" created
2020-08-19 13:04:21.858 30324-30805/pro.eventlive I/CCodec: Created component [c2.android.aac.encoder]
2020-08-19 13:04:21.858 30324-30805/pro.eventlive D/CCodecConfig: read media type: audio/mp4a-latm
2020-08-19 13:04:21.860 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: algo.buffers.max-count.values
2020-08-19 13:04:21.861 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: output.subscribed-indices.values
2020-08-19 13:04:21.861 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: input.buffers.allocator-ids.values
2020-08-19 13:04:21.861 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: output.buffers.allocator-ids.values
2020-08-19 13:04:21.861 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: algo.buffers.allocator-ids.values
2020-08-19 13:04:21.861 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: output.buffers.pool-ids.values
2020-08-19 13:04:21.862 30324-30805/pro.eventlive D/ReflectedParamUpdater: extent() != 1 for single value type: algo.buffers.pool-ids.values
2020-08-19 13:04:21.865 30324-30805/pro.eventlive I/CCodecConfig: query failed after returning 9 values (BAD_INDEX)
2020-08-19 13:04:21.865 30324-30805/pro.eventlive D/CCodecConfig: c2 config is Dict {
      c2::u32 coded.bitrate.value = 64000
      c2::u32 coded.pl.level = 0
      c2::u32 coded.pl.profile = 8192
      c2::u32 coding.aac-sbr-mode.value = 3
      c2::u32 input.buffers.max-size.value = 2048
      c2::u32 input.delay.value = 0
      string input.media-type.value = "audio/raw"
      string output.media-type.value = "audio/mp4a-latm"
      c2::u32 raw.channel-count.value = 1
      c2::u32 raw.sample-rate.value = 44100
    }
2020-08-19 13:04:21.866 30324-30805/pro.eventlive D/CCodecConfig: no c2 equivalents for aac-profile
2020-08-19 13:04:21.866 30324-30805/pro.eventlive D/CCodecConfig: no c2 equivalents for encoder
2020-08-19 13:04:21.868 30324-30805/pro.eventlive D/CCodecConfig: c2 config is Dict {
      c2::u32 coded.bitrate.value = 65536
      c2::u32 coded.pl.level = 0
      c2::u32 coded.pl.profile = 8192
      c2::u32 coding.aac-sbr-mode.value = 3
      c2::u32 input.buffers.max-size.value = 4096
      c2::u32 input.delay.value = 0
      string input.media-type.value = "audio/raw"
      string output.media-type.value = "audio/mp4a-latm"
      c2::u32 raw.channel-count.value = 2
      c2::u32 raw.sample-rate.value = 32000
    }
2020-08-19 13:04:21.869 30324-30805/pro.eventlive W/Codec2Client: query -- param skipped: index = 1107298332.
2020-08-19 13:04:21.869 30324-30805/pro.eventlive D/CCodec: setup formats input: AMessage(what = 0x00000000) = {
      int32_t channel-count = 2
      int32_t max-input-size = 5120
      string mime = "audio/raw"
      int32_t sample-rate = 32000
    } and output: AMessage(what = 0x00000000) = {
      int32_t bitrate = 65536
      int32_t channel-count = 2
      int32_t level = 0
      int32_t max-bitrate = 65536
      string mime = "audio/mp4a-latm"
      int32_t profile = 2
      int32_t sample-rate = 32000
    }
2020-08-19 13:04:21.869 30324-30324/pro.eventlive I/AudioEncoder: prepared
2020-08-19 13:04:21.878 30324-30324/pro.eventlive I/VideoEncoder: started
2020-08-19 13:04:21.880 30324-30793/pro.eventlive I/MediaCodec: MediaCodec will operate in async mode
2020-08-19 13:04:21.884 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Loaded->Idle
2020-08-19 13:04:21.930 30324-30793/pro.eventlive I/MediaCodec: setCodecState state : 0
2020-08-19 13:04:21.931 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Idle->Executing
2020-08-19 13:04:21.932 30324-30324/pro.eventlive I/AudioEncoder: started
2020-08-19 13:04:21.933 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Executing
2020-08-19 13:04:21.934 30324-30805/pro.eventlive I/MediaCodec: MediaCodec will operate in async mode
2020-08-19 13:04:21.935 30324-30805/pro.eventlive W/Codec2Client: query -- param skipped: index = 1342179345.
2020-08-19 13:04:21.936 30324-30805/pro.eventlive W/Codec2Client: query -- param skipped: index = 2415921170.
2020-08-19 13:04:21.937 30324-30805/pro.eventlive D/CCodecBufferChannel: [c2.android.aac.encoder#175] Created input block pool with allocatorID 16 => poolID 17 - OK (0)
2020-08-19 13:04:21.939 30324-30805/pro.eventlive I/CCodecBufferChannel: [c2.android.aac.encoder#175] Created output block pool with allocatorID 16 => poolID 30 - OK
2020-08-19 13:04:21.939 30324-30805/pro.eventlive D/CCodecBufferChannel: [c2.android.aac.encoder#175] Configured output block pool ids 30 => OK
2020-08-19 13:04:21.940 30324-30805/pro.eventlive E/ion: ioctl c0044901 failed with code -1: Invalid argument
2020-08-19 13:04:21.940 30324-30805/pro.eventlive E/C2AllocatorIon: >>>SEC C2AllocatorIon::mapUsage ion_is_legacy++++++++
2020-08-19 13:04:21.940 30324-30805/pro.eventlive E/C2AllocatorIon: >>>SEC mapUsage	0, 2000000
2020-08-19 13:04:21.943 30324-30324/pro.eventlive I/OpenGlViewBase: Thread started.
2020-08-19 13:04:21.972 30324-30324/pro.eventlive I/MicrophoneManager: Microphone started
2020-08-19 13:04:21.990 30324-30791/pro.eventlive I/CameraManagerGlobal: Camera 0 facing CAMERA_FACING_BACK state now CAMERA_STATE_OPEN for client pro.eventlive API Level 2
2020-08-19 13:04:22.029 30324-30324/pro.eventlive D/EvLive: Path to File: /storage/emulated/0/Android/data/pro.eventlive/files/Movies/EventLive/EventLive_20200819_130421.mp4
2020-08-19 13:04:22.030 30324-30849/pro.eventlive I/SrsFlvMuxer: worker: connecting to RTMP server by url=rtmp://go.eventlive.pro/live/KYMSJNUE
2020-08-19 13:04:22.030 30324-30849/pro.eventlive D/RtmpConnection: connect() called. Host: go.eventlive.pro, port: 1935, appName: live, publishPath: KYMSJNUE
2020-08-19 13:04:22.035 30324-30815/pro.eventlive I/Camera2ApiManager: Camera opened
2020-08-19 13:04:22.039 30324-30815/pro.eventlive I/Camera2ApiManager: Camera configured
2020-08-19 13:04:22.044 30324-30324/pro.eventlive I/VideoEncoder: stopped
2020-08-19 13:04:22.046 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Executing->Idle
2020-08-19 13:04:22.055 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Idle->Loaded
2020-08-19 13:04:22.055 30324-30794/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Loaded
2020-08-19 13:04:22.055 30324-30794/pro.eventlive I/ACodec:  [OMX.qcom.video.encoder.avc] Now kWhatShutdownCompleted event : 8513
2020-08-19 13:04:22.061 30324-30794/pro.eventlive I/ACodec:  [OMX.qcom.video.encoder.avc] Now uninitialized
2020-08-19 13:04:22.061 30324-30794/pro.eventlive I/ACodec:  [] Now kWhatShutdownCompleted event : 8513
2020-08-19 13:04:22.062 30324-30793/pro.eventlive I/MediaCodec: Codec shutdown complete
2020-08-19 13:04:22.067 30324-30324/pro.eventlive I/VideoEncoder: VideoEncoder OMX.qcom.video.encoder.avc
2020-08-19 13:04:22.067 30324-30324/pro.eventlive I/VideoEncoder: Color supported: 2141391876
2020-08-19 13:04:22.067 30324-30324/pro.eventlive I/VideoEncoder: Color supported: 2130708361
2020-08-19 13:04:22.068 30324-30324/pro.eventlive I/ACodec:  [] Now uninitialized
2020-08-19 13:04:22.069 30324-30859/pro.eventlive I/ACodec: [] onAllocateComponent
2020-08-19 13:04:22.070 30324-30859/pro.eventlive I/OMXClient: IOmx service obtained
2020-08-19 13:04:22.079 30324-30859/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] Now Loaded
2020-08-19 13:04:22.080 30324-30324/pro.eventlive I/VideoEncoder: Prepare video info: SURFACE, 1280x720
2020-08-19 13:04:22.081 30324-30859/pro.eventlive I/ACodec: app-pid(30324)
2020-08-19 13:04:22.082 30324-30859/pro.eventlive W/OMXUtils: do not know color format 0x7fa30c04 = 2141391876
2020-08-19 13:04:22.082 30324-30859/pro.eventlive W/OMXUtils: do not know color format 0x7f000789 = 2130708361
2020-08-19 13:04:22.090 30324-30859/pro.eventlive I/ACodec: app-name : pro.eventlive
2020-08-19 13:04:22.090 30324-30859/pro.eventlive I/ACodec: setupAVCEncoderParameters with [profile: Baseline] [level: Level31]
2020-08-19 13:04:22.091 30324-30859/pro.eventlive I/ACodec: Enable Perceptual Video Coding
2020-08-19 13:04:22.091 30324-30859/pro.eventlive I/ACodec: Success set IPB VideoMinQP(2/2/2) VideoMaxQP(50/50/50)
2020-08-19 13:04:22.091 30324-30859/pro.eventlive I/ACodec: reconfigEncoder4OtherApps
2020-08-19 13:04:22.093 30324-30859/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] cannot encode HDR static metadata. Ignoring.
2020-08-19 13:04:22.093 30324-30859/pro.eventlive I/ACodec: setupVideoEncoder succeeded
2020-08-19 13:04:22.093 30324-30859/pro.eventlive I/ACodec: [OMX.qcom.video.encoder.avc] configure, AMessage : AMessage(what = 'conf', target = 8) = {
      int32_t color-format = 2130708361
      int32_t i-frame-interval = 2
      string mime = "video/avc"
      int32_t width = 1280
      int32_t bitrate = 2048000
      int32_t max-input-size = 0
      int32_t frame-rate = 30
      int32_t height = 720
      int32_t encoder = 1
    }
2020-08-19 13:04:22.096 30324-30859/pro.eventlive W/OMXUtils: do not know color format 0x7f000789 = 2130708361
2020-08-19 13:04:22.108 30324-30324/pro.eventlive I/VideoEncoder: prepared
2020-08-19 13:04:22.108 30324-30324/pro.eventlive I/VideoEncoder: started
2020-08-19 13:04:22.119 30324-30849/pro.eventlive D/RtmpConnection: connect(): socket connection established, doing handhake...
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC0
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC1
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC1(): Calculating digest offset
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC1(): (real value of) digestOffset: 221
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC1(): recalculated digestOffset: 997
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: writeC1(): digestOffsetBytes: 000000DD
2020-08-19 13:04:22.120 30324-30849/pro.eventlive D/Handshake: partBeforeDigest(): size: 997
2020-08-19 13:04:22.121 30324-30849/pro.eventlive D/Handshake: writeC1(): Writing timestamp and Flash Player version
2020-08-19 13:04:22.121 30324-30849/pro.eventlive D/Handshake: partAfterDigest(): size: 507
2020-08-19 13:04:22.122 30324-30849/pro.eventlive D/Handshake: copying digest offset bytes in partBeforeDigest
2020-08-19 13:04:22.122 30324-30849/pro.eventlive D/Handshake: writeC1(): Calculating digest
2020-08-19 13:04:22.125 30324-30849/pro.eventlive D/Handshake: writeC1(): writing C1 packet
2020-08-19 13:04:22.125 30324-30849/pro.eventlive D/Handshake: readS0
2020-08-19 13:04:22.126 30324-30324/pro.eventlive D/DecorView: semSetRoundedCorners: 5
2020-08-19 13:04:22.189 30324-30849/pro.eventlive D/Handshake: readS1
2020-08-19 13:04:22.189 30324-30849/pro.eventlive D/Handshake: readS1(): S1 total bytes read OK
2020-08-19 13:04:22.189 30324-30849/pro.eventlive D/Handshake: writeC2
2020-08-19 13:04:22.189 30324-30849/pro.eventlive D/Handshake: readS2
2020-08-19 13:04:22.256 30324-30849/pro.eventlive D/Handshake: readS2(): S2 total bytes read OK
2020-08-19 13:04:22.256 30324-30849/pro.eventlive D/RtmpConnection: connect(): handshake done
2020-08-19 13:04:22.257 30324-30849/pro.eventlive D/RtmpConnection: rtmpConnect(): Building 'connect' invoke packet
2020-08-19 13:04:22.257 30324-30882/pro.eventlive D/RtmpConnection: starting main rx handler loop
2020-08-19 13:04:22.260 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Command (command: connect, transaction ID: 1), size: 270
2020-08-19 13:04:22.338 30324-30344/pro.eventlive I/CameraManagerGlobal: Camera 0 facing CAMERA_FACING_BACK state now CAMERA_STATE_ACTIVE for client pro.eventlive API Level 2
2020-08-19 13:04:22.339 30324-30882/pro.eventlive D/RtmpConnection: handleRxPacketLoop(): Setting acknowledgement window size: 5000000
2020-08-19 13:04:22.440 30324-30882/pro.eventlive D/RtmpConnection: handleRxPacketLoop(): Send acknowledgement window size: 1048576
2020-08-19 13:04:22.440 30324-30882/pro.eventlive D/RtmpConnection: wrote packet: RTMP Window Acknowledgment Size, size: 4
2020-08-19 13:04:22.441 30324-30882/pro.eventlive D/RtmpDecoder: readPacket(): Setting chunk size to: 8192
2020-08-19 13:04:22.444 30324-30882/pro.eventlive I/RtmpConnection: handleRxInvoke: Got result for invoked method: connect
2020-08-19 13:04:22.444 30324-30849/pro.eventlive D/RtmpConnection: createStream(): Sending releaseStream command...
2020-08-19 13:04:22.445 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Command (command: releaseStream, transaction ID: 2), size: 37
2020-08-19 13:04:22.448 30324-30849/pro.eventlive D/RtmpConnection: createStream(): Sending FCPublish command...
2020-08-19 13:04:22.448 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Command (command: FCPublish, transaction ID: 3), size: 33
2020-08-19 13:04:22.448 30324-30849/pro.eventlive D/RtmpConnection: createStream(): Sending createStream command...
2020-08-19 13:04:22.449 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Command (command: createStream, transaction ID: 4), size: 25
2020-08-19 13:04:22.650 30324-30882/pro.eventlive D/RtmpConnection: handleRxInvoke(): 'onFCPublish'
2020-08-19 13:04:22.751 30324-30882/pro.eventlive I/RtmpConnection: handleRxInvoke: Got result for invoked method: createStream
2020-08-19 13:04:22.751 30324-30882/pro.eventlive D/RtmpConnection: handleRxInvoke(): Stream ID to publish: 1
2020-08-19 13:04:22.751 30324-30882/pro.eventlive D/RtmpConnection: fmlePublish(): Sending publish command...
2020-08-19 13:04:22.752 30324-30882/pro.eventlive D/RtmpConnection: wrote packet: RTMP Command (command: publish, transaction ID: 0), size: 38
2020-08-19 13:04:23.194 30324-30811/pro.eventlive I/SrsFlvMuxer: frame discarded
2020-08-19 13:04:23.197 30324-30811/pro.eventlive I/SrsFlvMuxer: frame discarded
2020-08-19 13:04:23.199 30324-30811/pro.eventlive I/SrsFlvMuxer: frame discarded
2020-08-19 13:04:23.204 30324-30811/pro.eventlive I/chatty: uid=10244(pro.eventlive) BaseEncoder identical 2 lines
2020-08-19 13:04:23.205 30324-30811/pro.eventlive I/SrsFlvMuxer: frame discarded
2020-08-19 13:04:23.252 30324-30882/pro.eventlive D/RtmpConnection: handleRxInvoke(): onStatus NetStream.Publish.Start
2020-08-19 13:04:23.252 30324-30882/pro.eventlive D/RtmpConnection: onMetaData(): Sending empty onMetaData...
2020-08-19 13:04:23.255 30324-30882/pro.eventlive D/RtmpConnection: wrote packet: com.github.faucamp.simplertmp.packets.Data@b099232, size: 284
2020-08-19 13:04:23.258 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 11
2020-08-19 13:04:23.263 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 264
2020-08-19 13:04:23.265 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 264
2020-08-19 13:04:23.268 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 347
2020-08-19 13:04:23.271 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 353
2020-08-19 13:04:23.274 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 297
2020-08-19 13:04:23.277 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 329
2020-08-19 13:04:23.279 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 308
2020-08-19 13:04:23.281 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 301
2020-08-19 13:04:23.283 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 290
2020-08-19 13:04:23.285 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 289
2020-08-19 13:04:23.287 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 286
2020-08-19 13:04:23.288 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 243
2020-08-19 13:04:23.290 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 251
2020-08-19 13:04:23.292 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 259
2020-08-19 13:04:23.294 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 267
2020-08-19 13:04:23.296 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 299
2020-08-19 13:04:23.298 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 275
2020-08-19 13:04:23.299 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 264
2020-08-19 13:04:23.301 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 278
2020-08-19 13:04:23.303 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 287
2020-08-19 13:04:23.305 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 300
2020-08-19 13:04:23.307 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 259
2020-08-19 13:04:23.308 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 259
2020-08-19 13:04:23.310 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 274
2020-08-19 13:04:23.312 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 296
2020-08-19 13:04:23.314 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 247
2020-08-19 13:04:23.315 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 258
2020-08-19 13:04:23.317 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 292
2020-08-19 13:04:23.319 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 295
2020-08-19 13:04:23.388 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 261
2020-08-19 13:04:23.423 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 252
2020-08-19 13:04:23.466 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 250
2020-08-19 13:04:23.505 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 289
2020-08-19 13:04:23.507 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 263
2020-08-19 13:04:23.508 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 266
2020-08-19 13:04:23.510 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 276
2020-08-19 13:04:23.512 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 281
2020-08-19 13:04:23.514 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 244
2020-08-19 13:04:23.667 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 243
2020-08-19 13:04:23.704 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 245
2020-08-19 13:04:23.743 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 256
2020-08-19 13:04:23.781 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 265
2020-08-19 13:04:23.824 30324-30849/pro.eventlive D/RtmpConnection: wrote packet: RTMP Audio, size: 297
```

**Comment by pedroSG94** [comment]
My bad. Fixed in 1.8.7: 
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/08049168c893ee6ee88d233e0dea47be7b3f7f01

**Comment by mkrn** [comment]
Thanks, this may be closed now !
